### PR TITLE
feat: add mol and sdf preview support

### PR DESCRIPTION
**Pull request by khyurri**

### CODE REVIEW

File: web/src/api/hooks/api.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ type BadgerFetchOptions = {
     withCredentials?: boolean;
     plainHeaders?: boolean;
     isBlob?: boolean;
+    isText?: boolean; // todo: isBlob and isText must be changed to 1 field type
```
Comment (MikhailFokanov) [inline web/src/api/hooks/api.ts:27]: I know that you want to change this in future, but my sense of perfection suffers from it today...

File: web/src/shared/components/document-pages/document-pages.tsx [hunk 2]
```diff
@@ -1,24 +1,25 @@
 // temporary_disabled_rules
 /* eslint-disable @typescript-eslint/no-redeclare, react-hooks/exhaustive-deps */
-import React, { Fragment, useEffect, useState, useRef, useCallback, useMemo } from 'react';
+import { Spinner } from '@epam/loveship';
```
Comment (MikhailFokanov) [inline web/src/shared/components/document-pages/document-pages.tsx:3]: Maybe you could revert unnecessary changes and use the project's ordering of imports (in code style) in order not to format a lot of code.

**Review by MikhailFokanov — APPROVED** [review]

File: web/src/shared/components/document-pages/document-pages.tsx (hunk 2 above)
Comment (khyurri) [reply web/src/shared/components/document-pages/document-pages.tsx:3]: As I can see `prettier` in pre-commit requires such changes. Don't know who and how enabled it, but I just follow existing rules

File: web/src/api/hooks/api.ts (hunk 1 above)
Comment (khyurri) [reply web/src/api/hooks/api.ts:27]: I didn't refactor `isBlob` field just to skip testing almost whole project. Searching by `useBadgerFetch` returns 98 results in 21 files

**Comment by khyurri** [comment]
Thank you for review, @MikhailFokanov
